src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -191,22 +190,28 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
 
   const titleString = `${artist.name} - Auction Results on Artsy`
 
+  const handleMarketStatsRendered = () => {
+    // Scroll to auction results if param flag is present
+    if (!scrollToAuctionResults) return
+
+    setImmediate(() => scrollToAuctionResultsTop(), 0)
```

Comment:
Waiting until market stats are loaded to avoid a layout shift after scrolling which could result in a wrong scroll offset depending on the market section beeing rendered or not.